messaging/WFLY-12243_monitor_artemis_resource_adapter_thread_pools.adoc

```diff
@@ -0,0 +1,115 @@
+* Add a integer `global-client-scheduled-thread-pool--current-thread-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a long (in nanoseconds) `global-client-scheduled-thread-pool-keepalive-time` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a integer `global-client-scheduled-thread-pool-largest-thread-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a long `global-client-scheduled-thread-pool-task-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a integer `global-client-thread-pool-active-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a long `global-client-thread-pool-completed-task-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a integer `global-client-thread-pool-current-thread-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a long (in nanoseconds) `global-client-thread-pool-keepalive-time` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a integer `global-client-thread-pool-largest-thread-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+* Add a long `global-client-thread-pool-task-count` metric to https://wildscribe.github.io/WildFly/17.0/subsystem/messaging-activemq/[/subsystem=messaging-activemq]
+
+
+CLI to read the metrics:
+
+----
+[standalone@localhost:9990 /] /subsystem=messaging-activemq:read-resource(include-runtime)
+{
+    "outcome" => "success",
+    "result" => {
+        "global-client-scheduled-thread-pool-active-count" => 0,
+        "global-client-scheduled-thread-pool-completed-task-count" => 0L,
+        "global-client-scheduled-thread-pool-current-thread-count" => 0,
+        "global-client-scheduled-thread-pool-keepalive-time" => 10000000L,
+        "global-client-scheduled-thread-pool-largest-thread-count" => 0,
+        "global-client-scheduled-thread-pool-max-size" => undefined,
+        "global-client-scheduled-thread-pool-task-count" => 0L,
+        "global-client-thread-pool-active-count" => 0,
+        "global-client-thread-pool-completed-task-count" => 2L,
+        "global-client-thread-pool-current-thread-count" => 2,
+        "global-client-thread-pool-keepalive-time" => 60000000000L,
+        "global-client-thread-pool-largest-thread-count" => 2,
+        "global-client-thread-pool-max-size" => undefined,
```

Comment:
I do not see thread-pool-max-size in requirements. Has it been already implemented?

Reply:
global-client-scheduled-thread-pool-max-size is already existing and is a configuration parameter to define the max pool size